rental/views.py

```diff
@@ -94,27 +112,69 @@ def hardware_location(request):
 
   return render(request,'hardware_location.html', {'inventories': inventories, 'free_inventories': free_inventories})
 
-def rent_device(request):
+def reserve_device(request):
   user = User.objects.get(username=request.GET['user'])
   event = Event.objects.get(name=request.GET['event'])
   device = Device.objects.get(name=request.GET['device'])
-  device_rentals = Rental.objects.filter(inventory__device=device, returned = False)
-  free_inventory = Inventory.objects.filter(Q(event=event,device=device) & ~Q(rental=device_rentals)) 
+  rentals = Rental.objects.filter(event = event, device = device, returned = False, reservation = False)
```

Comment:
Rental does not have a device field,  it has an inventory, which has a device field. That's why I had 

Rental.objects.filter(inventory__device=device, returned = False) previously